builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -146,7 +145,7 @@ node() {
   def isEventSchdld = false
   def internalAccess = config['require_internal_access']
   domain = config['domain']
-  def roleARN = configLoader.AWS.ROLEID.replaceAll("/", "\\\\/")
+  def roleARN = configLoader.AWS.PLATFORMSERVICES_ROLEID.replaceAll("/", "\\\\/")
```

Comment:
In develop branch we are using roleId, are you sure about this change?

Reply:
as part of multi-account changes, roleid has been replaced by 

- "USERSERVICES_ROLEID": "arn:aws:iam::xxxxx:role/multienv0304_basic_execution"

- "PLATFORMSERVICES_ROLEID": "arn:aws:iam::xxxxx:role/multienv0304_platform_services"

Reply:
ok